plugins/channelrx/demodadsb/adsbdemodsink.h

```diff
@@ -25,10 +25,9 @@
 #include "dsp/channelsamplesink.h"
 #include "dsp/nco.h"
 #include "dsp/interpolator.h"
-#include "util/movingaverage.h"
+#include "util/message.h"
```

Comment:
There is no reference of the Message class in either the .h or the corresponding .cpp so I do not see the point of this

Reply:
I ha ta add `#include "util/message.h"` also in `adsbdemodsinkworker.h`